Client/Frontend/Intro/IntroViewModelV2.swift

```diff
@@ -27,6 +27,10 @@ class IntroViewModelV2 {
     // Initializer
     init() {
         onboardingResearch = OnboardingUserResearch()
-        screenType = onboardingResearch?.onboardingScreenType
+        // Case: Older user who has updated to a newer version and is not a first time user
+        // When user updates from a version of app which didn't have the
+        // user research onboarding screen type and is trying to Always Show the screen
+        // from Show Tour, we default to our .version1 of the screen
+        screenType = onboardingResearch?.onboardingScreenType ?? .versionV1
```

Comment:
Instead of adding guard and making checks to the BVC, I have mode the logic to the intro viewmodel where it actually belongs and also addresses the case for an update that older user makes to the new version.